database/seeders/EventsErstiwocheSeeder.php

```diff
@@ -363,25 +356,24 @@ public function runSport(): void
         // create event slots
         $slots = [
             [
-                'name' => 'Fußball, Volleyball, Basketball',
+                'name' => 'Hochschulsport',
```

Comment:
Please use `Fußball, Volleyball, Basketball` as name and not `Hochschulsport`, this is easier to understand for our first semester students.